include/El/lattice.h

```diff
@@ -52,6 +52,9 @@ typedef struct
     ElLLLVariant variant;
     bool recursive;
     ElInt cutoff;
+    float precisionFudge;
+    ElInt minColThresh;
+    bool unsafeSzReduct;
```

Comment:
This convention isn't written down anywhere, but I would recommend `unsafeSizeReduce` over `unsafeSzReduct`, as it is only one more letter and uses full words